### PR TITLE
added logging pipelines

### PR DESCRIPTION
**Pull request by greermichael**
- Incremented version of pipeline reference to 0.7.0
- Added create/remove pipelines for logging deployments (operations & application)

### CODE REVIEW

File: jenkins/jenkins.yaml.tpl [hunk 1]
```diff
@@ -534,13 +534,105 @@ jobs:
...
+                }
+                branch("0.7.0")
+              }
+            }
+            scriptPath("operations/logging/install/Jenkinsfile")
+          }
+        }
+       }
+      pipelineJob("LMA/Remove_Logging_Operations") {
+        displayName('Remove Logging in Operations Account (could impact logging in Application acct)')
```
Comment (radepal) [inline jenkins/jenkins.yaml.tpl]: (could impact logging in Application acct) move this comment to description

File: jenkins/jenkins.yaml.tpl [hunk 2]
```diff
@@ -534,13 +534,105 @@ jobs:
...
+                }
+                branch("0.7.0")
+              }
+            }
+            scriptPath("operations/logging/destroy/Jenkinsfile")
+          }
+        }
+       }
+      pipelineJob("LMA/Create_Logging_Application") {
+        displayName('Create Logging in Application Account (requires logging in operations first')
```
Comment (radepal) [inline jenkins/jenkins.yaml.tpl]: (requires logging in operations first) move this comment to description
missing ) at end

File: jenkins/jenkins.yaml.tpl (hunk 1 above)
Comment (greermichael) [reply jenkins/jenkins.yaml.tpl]: done

File: jenkins/jenkins.yaml.tpl (hunk 2 above)
Comment (greermichael) [reply jenkins/jenkins.yaml.tpl]: done

**Pull request merged**